lib/go-tc/servers.go

```diff
@@ -1019,6 +1021,12 @@ func UpdateServerPropertiesV40(profileNames []string, properties CommonServerPro
 	}
 }
 
+// ServerV41 is the representation of a Server in version 4.1 of the Traffic Ops API.
+type ServerV41 struct {
+	ServerV40
+	ASNs pq.Int32Array `json:"asns"`
```

Comment:
While I'm here I just want to mention that this should probably just be a `[]int` and you wrap the reference to the property with `pq.Int32Array` for scanning purposes. Makes consumption of the API much simpler.